pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1447,6 +1451,22 @@ func (c *AWSCloud) GetVolumeLabels(volumeName string) (map[string]string, error)
 	return labels, nil
 }
 
```

Comment:
The method `func (self *awsInstance) getMountDevice(...)` in this file appears to compute the location the device path for `AttachDisk()` calls. It looks at the local file system to figure this out. If Attach is being called from controller, i.e. different machine than where it is being attached, this will be broken right?